stack.cabal

```diff
@@ -219,7 +219,7 @@ library
                    , vector-binary-instances
                    , yaml >= 0.8.10.1
                    , zlib >= 0.5.4.2 && < 0.7
-                   , deepseq >= 1.4
+                   , deepseq >= 1.3 && < 1.5
```

Comment:
Yes, I saw that extra-dep - however, there are packages in lts-2 that have an upper bound of 1.3 on deepseq, so it should be impossible to build stack with lts-2 and deepseq 1.4 at the same time. And stack indeed complains building itself with lts-2.22 and deepseq-1.4.1.2: I created a gist [here](https://gist.github.com/hesiod/4ada2c589b6f6a8613e7e4a14c24a161). In particular, take a look at [lines 465 to 664](https://gist.github.com/hesiod/4ada2c589b6f6a8613e7e4a14c24a161#file-stack-build-output-L465-L664).